File: JAVA/Program to print Pascal's triangle.java
public class Main {

  public static void main(String[] args) {
    int rows = 6, var = 1;

    for(int i = 0; i < rows; i++) {
      for(int space = 1; space < rows - i; ++space) {
        System.out.print("  ");
      }

      for(int j = 0; j <= i; j++) {
        if (j == 0 || i == 0)
         var = 1;
        else
         var = var * (i - j + 1) / j;

        System.out.printf("%4d", var);
      }

      System.out.println();
    }
  }
}

File: Python/Program to print Pascal's Triangle.py
NumOfRows = int(input("Enter number of rows: "))
coeffcient = 1

for i in range(1, NumOfRows+1):
    for space in range(1, NumOfRows-i+1):
        print(" ",end="")
    for j in range(0, i):
        if j==0 or i==0:
            coeffcient = 1
        else:
            coeffcient = coeffcient * (i - j)//j
        print(coeffcient, end = " ")
    print()